docs/serving/configuration/config-defaults.md

```diff
@@ -30,319 +32,497 @@ data:
-Below is an example of setting revision-memory-limit.
-By default, it is not set by Knative.
+The `revision-memory-limit` value determines the default memory allocation limit for revisions. If this value is omitted, the system default is used. This key is not enabled by default for Knative.
 {% endraw %}
 
-**Key**: `revision-memory-limit`
-
-**Default**: `"200M"` (200 megabytes of memory)
+* **Global key:** `revision-memory-limit`
+* **Per-revision annotation key:** `serving.knative.dev/revision-memory-limit`
+* **Possible values:** integer
+* **Default:** `"200M"` (200 megabytes of memory)
 
 **Example:**
-```yaml
-apiVersion:  v1
-kind:  ConfigMap
-metadata:
-  name:  config-defaults
-  namespace:  knative-serving
-data:
-  revision-memory-limit: "200M" # 200 megabytes of memory
-```
+
+=== "Global (ConfigMap)"
+    ```yaml
+    apiVersion:  v1
+    kind:  ConfigMap
+    metadata:
+      name:  config-defaults
+      namespace:  knative-serving
+    data:
+      revision-memory-limit: "200M"
+    ```
+
+=== "Per Revision"
+    ```yaml
+    apiVersion: serving.knative.dev/v1
+    kind: Service
+    metadata:
+      name: helloworld-go
+      namespace: default
+    spec:
+      template:
+        metadata:
+          annotations:
+            serving.knative.dev/revision-memory-limit: "200M"
+        spec:
+          containers:
+            - image: gcr.io/knative-samples/helloworld-go
+    ```
 
 ### Revision Ephemeral Storage Limit
 {% raw %}
-revision-ephemeral-storage-limit contains the ephemeral storage
-allocation to limit revisions to by default.  If omitted, no value is
-specified and the system default is used.
+The `revision-ephemeral-storage-limit` value determines the default ephemeral storage limit allocated to revisions. If this value is omitted, the system default is used. This key is not enabled by default for Knative.
 {% endraw %}
 
-**Key**: `revision-ephemeral-storage-limit`
-
-**Default**: `"750M"` (750 megabytes of storage)
+* **Global key:** `revision-ephemeral-storage-limit`
+* **Per-revision annotation key:** `serving.knative.dev/revision-ephemeral-storage-limit`
+* **Possible values:** integer
+* **Default:** `"750M"` (750 megabytes of storage)
 
 **Example:**
-```yaml
-apiVersion:  v1
-kind:  ConfigMap
-metadata:
-  name:  config-defaults
-  namespace:  knative-serving
-data:
-  revision-ephemeral-storage-limit: "750M" # 750 megabytes of storage
-```
 
-### Container Name Template
+=== "Global (ConfigMap)"
+    ```yaml
+    apiVersion:  v1
+    kind:  ConfigMap
+    metadata:
+      name:  config-defaults
+      namespace:  knative-serving
+    data:
+      revision-ephemeral-storage-limit: "750M"
+    ```
+
+=== "Per Revision"
+    ```yaml
+    apiVersion: serving.knative.dev/v1
+    kind: Service
+    metadata:
+      name: helloworld-go
+      namespace: default
+    spec:
+      template:
+        metadata:
+          annotations:
+            serving.knative.dev/revision-ephemeral-storage-limit: "750M"
```

Comment:
Likewise this annotation doesn't exist - the config map property configures the default value for `Spec.Containers[*].Limits.EphemeralStorage` of a `Revision`

Reply:
see previous comments

Reply:
similar to above